src/sentry/conf/server.py

```diff
@@ -522,6 +522,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "sentry.tasks.assemble",
     "sentry.tasks.auth",
     "sentry.tasks.auto_resolve_issues",
+    "sentry.tasks.auto_review_inbox",
```

Comment:
Nit: Should this be `auto_remove_inbox`? Or something along the lines of `purge_stale_inbox_groups`?